app/i18n/cs.ts

```diff
@@ -0,0 +1,185 @@
+import { I18nLocale } from '.'
+
+const pluralizeForms: Record<string, string[]> = {
+  serving: ['porce', 'porce', 'porcí'],
+  day: ['den', 'dny', 'dní'],
+  adult: ['dospělý', 'dospělí', 'dospělých'],
+  kid: ['dítě', 'děti', 'dětí'],
+  roll: ['role', 'role', 'rolí']
+}
+
+const ruLocale: I18nLocale = {
+  localeName: 'Česky',
+
+  title: 'Генератор списка покупок во время пандемии коронавируса',
+  description:
+    'Что покупать и готовить во время пандемии коронавируса? Как пережить карантин? Укажите, насколько большая ваша семья, что вы предпочитаете есть и мы составим для вас рациональный список покупок.',
+
+  about: {
+    collapsed: `<strong>Для чего это приложение?</strong>`,
+    expanded: `
+<p>
+  <strong>Для чего это приложение?</strong> Используйте генератор для планирования покупок во время пандемии коронавируса. Выберите продолжительность пребывания, количество человек, меню и получите список продуктов для покупки.
+</p>
+
+<p>
+  Используйте функцию «Поделиться списком», чтобы создать список задач и поделиться им со своей семьей или отправить на телефон.
+</p>
+
+<p>
+  <a href="mailto:koss@nocorp.me">Напишите мне</a>, если у вас есть какие-либо отзывы!
+</p>
+`
+  },
+
+  measurement: {
+    system: 'Система мер',
+    metric: 'Метрическая',
+    imperial: 'Имперская'
+  },
+  numberOfDays: 'Продолжительность в днях',
+  numberOfAdults: 'Кол-во взрослых',
+  numberOfKids: 'детей',
+  kidsServing: {
+    title: 'Детская порция рассчитывается как',
+    adult: 'взрослая порция'
+  },
+
+  sections: {
+    essentials: {
+      title: 'Предметы первой необходимости'
+    },
+
+    breakfast: {
+      title: 'Завтрак'
+    },
+
+    meals: {
+      title: 'Обеды и ужины',
+      formula: {
+        intro: '2 блюда в день'
+      }
+    },
+
+    drinks: {
+      title: 'Напитки',
+      description: 'Три напитка в день на человека. На завтрак, обед и ужин.'
+    },
+
+    preview: {
+      title: 'Список покупок',
+      share: 'Поделиться списком'
+    },
+
+    list: {
+      title: 'Список покупок',
+      description:
+        'Поделитесь этой страницей со своей семьей. Когда вы отмечаете пункт как купленный, список автоматически обновляется на всех устройствах с открытой страницей.',
+      generated: {
+        intro: 'Этот список сгенерировать с помощью',
+        link: 'генератора списка покупок во время пандемии коронавируса'
+      }
```

Comment:
Please also add (after generated):

```
addToList: 'Добавить в список',
remove: 'Удалить'
```

I've added the ability to add custom items to the list.